### DISCUSSION TITLE
course/03

### DISCUSSION THREAD

**Discussion by giscus** [mitre/inspec-advanced-developer#9] [Ideas]
# course/03

The MITRE Security Automation Framework Team's Advanced InSpec profile development course

https://mitre-inspec-advanced-developer.netlify.app/course/03.html

**Comment by em-c-rod** [comment]
Add a prerequisite to the class that is a small tutorial on using `pwd` `ls` `cd` and simple file structures.

**Reply by em-c-rod** [reply]
Consider adding this as a step for people to do while they are waiting for everyone to join the class in the beginning.
